sonar-kotlin-plugin/src/main/java/org/sonarsource/kotlin/api/FilteredAbstractCheck.kt

```diff
@@ -0,0 +1,16 @@
+package org.sonarsource.kotlin.api
+
+import org.jetbrains.kotlin.psi.KtCallExpression
+import org.jetbrains.kotlin.resolve.calls.callUtil.getResolvedCall
+import org.jetbrains.kotlin.resolve.calls.model.ResolvedCall
+import org.sonarsource.kotlin.plugin.KotlinFileContext
+
+abstract class FilteredAbstractCheck : AbstractCheck() {
```

Comment:
This name doesn't reflect the purpose of this class. Maybe `CallExpressionsAbstractCheck` or `CallExpressionsVisitor`, or I believe you'll come up with a better one :)